zipline-kotlin-plugin/src/main/kotlin/app/cash/zipline/kotlin/AdapterGenerator.kt

```diff
@@ -82,70 +79,40 @@ internal class AdapterGenerator(
 
   private val bridgedInterface = BridgedInterface.create(
     pluginContext,
-    messageCollector,
     ziplineApis,
     scope,
     original,
     "Zipline.take()",
     original.defaultType
   )
 
-  /** Returns an expression that references the adapter, creating it if necessary. */
+  /** Returns an expression that references the adapter, generating it if necessary. */
   fun adapterExpression(type: IrSimpleType): IrExpression {
     val adapterClass = generateAdapterIfAbsent()
     val irBlockBodyBuilder = irBlockBodyBuilder(pluginContext, scope, original)
     return irBlockBodyBuilder.adapterExpression(adapterClass, type)
   }
 
+  /**
+   * Given a type like `GenericEchoService<String, Long>`, this generates a call to the constructor
+   * of that type.
+   */
   private fun IrBuilderWithScope.adapterExpression(
     adapterClass: IrClass,
     adapterType: IrSimpleType,
   ): IrExpression {
-    // Given a declaration like this: interface SampleService<K, V> : ZiplineService
-    //     and an instance like this: SampleService<String, Long>
-    // Creates a map from the type parameters to their concrete types.
-    val substitutionMap = mutableMapOf<IrTypeParameterSymbol, IrType>()
-    for (i in original.typeParameters.indices) {
-      substitutionMap[original.typeParameters[i].symbol] = adapterType.arguments[i] as IrType
-    }
-
     // listOf(
-    //   typeOf<SampleRequest<String>>(),
-    //   typeOf<SampleResponse<Long>>(),
+    //   serializer<String>(),
```

Comment:
Is this example code or commented out?

Reply:
Example code, attempting to demonstrate what it’s generating